### DISCUSSION TITLE
关于登录验证码的建议

### DISCUSSION THREAD

**Discussion by lele0124** [flipped-aurora/gin-vue-admin#319] [General]
现在登录验证码，通过第三方插件生成和验证，如果多台服务器部署情况下，如果nginx 没有配置策略，会出现验证码无法验证的情况（验证码生成和验证不在同一台服务器上），能否把验证码生成的配置文件放置到config（global.GVA_CONFIG.Captcha.ImgHeight, global.GVA_CONFIG.Captcha.ImgWidth, global.GVA_CONFIG.Captcha.KeyLong, 0.7, 80）
，把生成的验证码设置过期时效后，存入到redis中，统一到redis认证，这样方便集群部署。

**Comment by SliverHorn** [comment]
- 你可以看看这个pr的 #300 
- 需要注意的点是1.代码可行性，2. 注意需要redis的初始化
